loops_lab.js

```diff
@@ -0,0 +1,270 @@
+// # Loops Lab
+//
+// To complete this lab, do the following:
+//
+// 1. Fork this repo
+// 2. Clone **your** forked repo
+// 3. Create a file called `loops_lab.js` inside your cloned repo
+// 4. Copy the description of the exercise as a comment
+// 5. Underneath the commented description write your code.
+// 6. Ensure your solutions work properly!
+// 7. Open PR against this repo.
+//
+// ___
+//
+// ## Exercises
+//
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+
+let num = 10;
+
+while (num > 0) {
+  console.log("1.a " + num);
+  num--;
+}
+
+let num2 = 10;
+for(num2; num2 > 0; num2--) {
```

Comment:
It's better practice to declare variable with let inside for loops instead of outside in order to avoid case where you might unintentionally change a value of a variable. For Example: 

let num = "lalala"
for (num = 8; num > 1; --num) {
if (num === 8) {
num--;
}
console.log(num);
}

console.log("what is num?", num) <== num will become 1 instead of "lalala"